initDB/build_db.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE account(
```

Comment:
I believe you need to init or create the database first?

Reply:
The Dockerfile initializes the database and "uses" it (line 3). I've already ran a container made with the image and it has a database named `lugdnd` with the `account` table inside it.

Reply:
ok I'm approving